### ISSUE TITLE
Install documentation improvement

### ISSUE THREAD

**Issue by guziy** [Toblerity/Shapely#214]
Hi:

It would help users who want to use geos, but they do not have permissions to install it to the usual places (like me). So if your geos is installed to $GEOS_DIR, the following command will help shapely to find it:

```
CFLAGS="-I${GEOS_DIR}/include" pip install shapely -I
```

Cheers

**Comment by sgillies** [comment]
@guziy or how about using geos-config instead?

```
CFLAGS=`geos-config --cflags` LDFLAGS=`geos-config --clibs` pip install shapely
```

**Comment by sgillies** [comment]
Closed via fa1f436

**Comment by guziy** [comment]
That is much better.

I did not have to specify LDFLAGS but that is probably because I have it in
LD_LIBRARY_PATH.

Thanks

2015-01-04 17:23 GMT-05:00 Sean Gillies <notifications@github.com>:

> @guziy <https://github.com/guziy> or how about using geos-config instead?
>
> CFLAGS=`geos-config --cflags` LDFLAGS=`geos-config --clibs` pip install shapely
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Toblerity/Shapely/issues/214#issuecomment-68652208>.
>



-- 
Sasha
